### PR TITLE
DP-23356: Bulk parent update

### PR DESCRIPTION
**Pull request by tom-fleming**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-23356

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/modules/custom/mass_admin_pages/mass_admin_pages.services.yml [hunk 1]
```diff
@@ -4,3 +4,8 @@ services:
     tags:
       - {name: event_subscriber}
     arguments: ['@datetime.time', '@router.admin_context']
+
+  theme.negotiator.mass_admin_pages:
+    class: Drupal\mass_admin_pages\Theme\ThemeNegotiator
+    tags:
+      - { name: theme_negotiator, priority: -50 }
```
Comment (weekbeforenext) [inline docroot/modules/custom/mass_admin_pages/mass_admin_pages.services.yml:11]: Probably need a blank line at the end of this file.

File: patches/DP-23411_node-hierarchy-async-support-save-only-changed.patch [hunk 2]
```diff
@@ -247,6 +247,19 @@ index fc94005..87419e8 100644
...
+diff --git a/src/Plugin/Derivative/DynamicLocalTasks.php b/src/Plugin/Derivative/DynamicLocalTasks.php
+index bc8bd8a..e065096 100644
+--- a/src/Plugin/Derivative/DynamicLocalTasks.php
++++ b/src/Plugin/Derivative/DynamicLocalTasks.php
+@@ -80,7 +80,7 @@ class DynamicLocalTasks extends DeriverBase implements ContainerDeriverInterface
+       }
+       $this->derivatives["$entity_type_id.entity_hierarchy_reorder"] = [
+         'route_name' => "entity.$entity_type_id.entity_hierarchy_reorder",
+-        'title' => $this->t('Children'),
++        'title' => $this->t('Hierarchy'),
```
Comment (weekbeforenext) [inline patches/DP-23411_node-hierarchy-async-support-save-only-changed.patch:259]: We might have been able to override this in a custom module but this patch works, too.

**Review by weekbeforenext — APPROVED** [review]
I think this looks good to go from code review perspective.

File: patches/DP-23411_node-hierarchy-async-support-save-only-changed.patch (hunk 2 above)
Comment (beto-aveiga) [reply patches/DP-23411_node-hierarchy-async-support-save-only-changed.patch:259]: @weekbeforenext I did try overriding it using a module. In fact, there is a file on this PR that should do this. However, it worked for the title on that route, but not for the local task when viewing the node. I spent at least 2 hours understanding why it wasn't changing, but I wasn't able to solve it 😞 ... So I add it to the patch.

File: docroot/modules/custom/mass_admin_pages/mass_admin_pages.services.yml (hunk 1 above)
Comment (beto-aveiga) [reply docroot/modules/custom/mass_admin_pages/mass_admin_pages.services.yml:11]: @weekbeforenext  sorry, I missed this comment before the merge.

**Pull request merged**